ssofact.module

```diff
@@ -35,23 +35,216 @@ function ssofact_help($route_name, RouteMatchInterface $route_match) {
 function ssofact_openid_connect_user_properties_ignore_alter(array &$properties_ignore, array $context) {
   unset($properties_ignore[array_search('mail', $properties_ignore)]);
   // @todo Triggers (logged) error: 'property type not implemented: changed'
-  //unset($properties_ignore[array_search('changed', $properties_ignore)]);
+  // unset($properties_ignore[array_search('changed', $properties_ignore)]);
 }
 
 /**
  * Implements hook_openid_connect_userinfo_save().
  */
 function ssofact_openid_connect_userinfo_save(UserInterface $account, $context) {
-  $userinfo = $context['userinfo'];
-  if (empty($userinfo['deactivated'])) {
+  $user_info = $context['userinfo'];
```

Comment:
Why did you change the name here?  The variable is named `$userinfo` in the rest of the code.  It's also not consistent with the key in `$context`.  Renaming it here is bad for discovery/code-search.